src/components/cylc/tree/cylc-tree.js

```diff
@@ -14,14 +14,39 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { computePercentProgress } from '@/components/cylc'
 import { extractGroupState } from '@/utils/tasks'
 import { merge } from 'lodash'
 import { createFamilyProxyNode, getCyclePointId } from '@/components/cylc/tree/index'
 import TaskState from '@/model/TaskState.model'
 
 export const FAMILY_ROOT = 'root'
 
+/**
+ * Compute percent progress.
+ *
+ * @see https://github.com/cylc/cylc-flow/blob/de7d938496e82dbdfb165938145670dd8e801efd/lib/cylc/gui/updater_tree.py#L248-L263
+ * @param {number|undefined} startedTime in milliseconds since 1970-01-01 00:00:00 UTC, e.g. 1568353099874
+ * @param {number|undefined} meanElapsedTime mean elapsed time in seconds
+ * @returns {number} the percent progress, e.g. 25 (meaning 25% progress)
+ * @private
+ */
+function computePercentProgress (startedTime, meanElapsedTime) {
+  // startedTime > now reportedly possible via interaction with `cylc reset`
```

Comment:
This comment belongs with the next code block?  In fact, `cylc reset` no longer exists (since SoD) so `startedTime > now` should not be possible anymore and we can remove the comment and the code.

Reply:
Good catch! Moved it down.

Reply:
(I think you missed the second sentence in my comment here :grin: )

Reply:
Sorry. I agree that can be removed if `cylc reset` has been removed. Not sure if that could happen due to time zone, difference in time or time precision between UI/browser & UI Server (as you pointed in Element).

What do you think @hjoliver ? If we remove this, we just need to move the `meanElapsedTime === 0` check up and return `0` to avoid division by zero.

Reply:
OK let's remove the comment about reset, but keep the if block in case of clock sync issues (and add a comment for that, so we know there's not some more nefarious reason that we need to be aware of in future).

Reply:
Roger, removed the part about `cylc reset`, tried to list a few possible cases that could cause an issue without that check.